target/linux/ramips/dts/MAESTRO-E220.dts

```diff
@@ -0,0 +1,112 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "maestro-e220", "ralink,mt7620a-soc";
+	model = "Maestro E220";
+    chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+	gpio-leds {
+		compatible = "gpio-leds";
+		power {
+			label = "maestro-e220:green:power";
+			gpios = <&gpio2 20 1>;
+        };
+		wifi {
+			label = "maestro-e220:blue:wifi";
+			gpios = <&gpio3 0 1>;
+		};
+		network {
+			label = "maestro-e220:orange:network";
+			gpios = <&gpio1 2 1>;
+		};
+		activity {
+			label = "maestro-e220:orange:activity";
+			gpios = <&gpio1 3 1>;
+		};
+		signal {
+			label = "maestro-e220:orange:signal";
+			gpios = <&gpio1 4 1>;
+		};
+		alert {
+			label = "maestro-e220:red:alert";
+			gpios = <&gpio1 5 1>;
+		};
+    };
+};
+
+&gpio0 {
+    status = "okay";
+};
+
+&gpio1 {
+    status = "okay";
+};
+
+&gpio2 {
+    status = "okay";
+};
+
+&spi0 {
+    status = "okay";
+
+    m25p80@0 {
+        #address-cells = <1>;
+        #size-cells = <1>;
+        compatible = "w25q256";
+        reg = <0 0>;
+        linux,modalias = "m25p80", "w25q256";
+        spi-max-frequency = <10000000>;
+        partition@0 {
+            label = "u-boot";
+            reg = <0x0 0x30000>;
+            read-only;
+        };
+        partition@30000 {
+            label = "u-boot-env";
+            reg = <0x30000 0x10000>;
+            read-only;
+        };
+        factory: partition@40000 {
+            label = "factory";
+            reg = <0x40000 0x10000>;
+            read-only;
+        };
+        partition@50000 {
+            label = "firmware";
+            reg = <0x50000 0xFB0000>;
+        };
+    };
+};
+
+&ehci {
```

Comment:
does this board really have usb? at least the manual doesn't mention an usb port.